recipes/clapack/build.sh

```diff
@@ -0,0 +1,13 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX}
+make
```

Comment:
I found a check in the `CMAKEFILE`, I will add it.
`make cmake_check_build_system`